.github/workflows/awsfulltest.yml

```diff
@@ -0,0 +1,43 @@
+name: nf-core AWS full size tests
+# This workflow is triggered on push to the master branch.
+# It runs the -profile 'test_full' on AWS batch
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
```suggestion
  push:
    branches:
      - master
```

Reply:
So if you would like the CI tests to pass, the AWS full tests should only be triggered in releases, if you want to give it a try before release, maybe let's just check it in the dev branch first. Sorry, I was a bit confused about this